.pipelines/templates/release.yml

```diff
@@ -25,7 +25,7 @@ stages:
       steps:
         - bash: |
             set -eu  # exit on error
-            imageVer=$(cat VERSION-IMAGE-REDACTOR)
+            imageVer=$(grep -m 1 version presidio-image-redactor/pyproject.toml | tr -s ' ' | tr -d '"' | tr -d "'" | cut -d' ' -f3)
```

Comment:
We now also have a version for presidio-structured. Maybe we can align presidio-image-redactor and presidio-structured to have the same version? Then we have the packages in prod (analyzer, anonymizer) and those in beta (structured, image redactor). WDYT?

Reply:
This is for MCR versioning, so we can align the presidio-structured later to be the same as image redactor for pypi

Reply:
ok sounds good